### PR TITLE
Wallet farms current value info tooltip

### PR DESCRIPTION
**Pull request by kolebjak**

### CODE REVIEW

File: src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx [hunk 1]
```diff
@@ -41,65 +42,76 @@ export const useFarmingPositionsTable = (data: FarmingPositionsTableData[]) => {
...
-              </Text>
-            )}
-          >
-            <DisplayValue value={row.original.position.valueDisplay} />
-          </DollarAssetValue>
-        </div>
-      ),
-    }),
-  ]
+  const columns = useMemo(
```
Comment (vkulinich-cl) [inline src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: Why did you add the useMemo hook with no dependencies?

Comment (kolebjak) [reply src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: It's not necessary to recreate column array for each table render. There are no external dependencies needed to create these columns and that's why dependency array is empty.

Comment (kolebjak) [reply src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: There is a note in tanstack doc: 

> Note: If defining columns inside a component, you should still try to give the column definitions a stable identity. This will help with performance and prevent unnecessary re-renders. Store the column definitions in either a useMemo or useState hook.

Comment (vkulinich-cl) [reply src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: okay, nice then. Just fix a lint issue caused by the no dependencies

Comment (kolebjak) [reply src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: @vkulinich-cl  Looks like there're no eslint erros even after rebase.

Comment (vkulinich-cl) [reply src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: React Hook useMemo has missing dependencies: 'accessor', 'isDesktop', and 't'. Either include them or remove the dependency array. I have this lint warning caused by empty hook dependencies

Comment (kolebjak) [reply src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx:45]: Got it, thanks 👍

**Review by vkulinich-cl — APPROVED** [review]
